src/widgets/Footer/Footer.tsx

```diff
@@ -1,16 +1,26 @@
+import React from 'react';
+
 import Contacts from './components/Contacts';
-import CustomerCare from './components/CustomerCare';
+import CustomerCareList from './components/CustomerCareList';
 import FollowUs from './components/FollowUs';
 import OrderProcessing from './components/OrderProcessing';
 import Policies from './components/Policies';
-import QuickLnks from './components/QuickLinks';
+
 import { Subscribe } from './components/Subscribe';
 
+import FooterComponent from './components/FooterComponent';
+import QuickLinksList from './components/QuickLinksList';
+
 export const Footer = () => {
   return (
     <div className="flex flex-col bg-secondary-10 px-5 pt-10 pb-4">
-      <CustomerCare />
-      <QuickLnks />
+      <h3 className="text-lg text-azaret font-medium mb-10 phone:hidden tablet:block">
```

Comment:
you should not specify phone device. You could write just hidden without "phone", because all styles are accepted for phone before laptop, tablet and so on were not mentioned.

Reply:
Ok